### ISSUE TITLE
Idea for Cracked Sand growth

### ISSUE THREAD

**Issue by keybounce** [ExtrabiomesXL/ExtrabiomesXL#50]
I love the idea of it spreading.

I like the idea of having to deal with it spreading.

I don't like the idea of it destroying a world.

Ahh ... I know. Edging.

The core of the wasteland contains CS. It spreads. But it spreads at different rates in different areas.

In the core, it will spread quickly if not completely removed.
In the outer region of wastelands, it spreads slower.
Outside of wastelands ...

Well, here's a problem. I would just say force rivers, beaches, or oceans around the wastelands, just like the current biome borders are done. But that doesn't work in Myst.

So what you need is for CS to move slowly outside of wasteland. Perhaps a function of distance from the wasteland. Enough that you can play "fight the CS" if you want, and don't have to worry about losing your world if you do not.

Decay is nice in Myst. But there are times that I want to play with landscapes, and I don't want decay. I have TF because I like the landscape as well as the new creatures and things; I don't want myst decay in it.

I made some unusual lands with Myst; I think much of the 1.7+ terrain fan-boys wants (not all) are satisfied with the unusual landscapes you can make there. But if I'm going for landscape, I don't want decay.

I can get lands with the EB biomes in Myst. I'm discovering that the generation in Myst is significantly different than normal overworld, and I may turn EB on in the overworld -- at least a test world -- to see what things look like there.

In other words: I don't want to have to disable wasteland for my world to be safe. And I want to, when I want to fight the wasteland, have it actually do something.

So it has to be that in the core of the wasteland, it grows quickly; in the outer regions it does not start but will grow, and outside it grows slower; the farther from the wasteland, the slower the growth until it is significantly slower than grass's regrowth rate (as you pointed out, dirt can't have any status stored, so the growth has to be slow enough for grass to dominate.)
